extensions/http(s)/lib/ace/http/handler.ex

```diff
@@ -143,16 +141,25 @@ defmodule Ace.HTTP.Handler do
   end
 
   def add_header(request = %{headers: headers}, :Host, location) do
-    [host, port] = case String.split(location, ":") do
-      [host, port] -> [host, :erlang.binary_to_integer(port)]
-      [host] -> [host, 80]
-    end
     headers = headers ++ [{"host", location}]
-    %{request | headers: headers, host: host, port: port}
+    %{request | headers: headers, authority: location}
   end
   def add_header(request = %{headers: headers}, key, value) do
     key = String.downcase("#{key}")
     headers = headers ++ [{key, value}]
     %{request | headers: headers}
   end
+
+  defp split_path(path_string) do
```

Comment:
There is a `Raxx.split_path` function. it's not documented but if you wanted could make use of that

Reply:
I will use it. Documented in a PR to Raxx.